wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -137,4 +180,36 @@ extension ZMUserSession: UserSession {
             }
         }
     }
+
+    static let MaxVideoWidth: UInt64 = 1920 // FullHD
+
+    static let MaxAudioLength: TimeInterval = 1500 // 25 minutes (25 * 60.0)
+    private static let MaxTeamAudioLength: TimeInterval = 6000 // 100 minutes (100 * 60.0)
+    private static let MaxVideoLength: TimeInterval = 240 // 4 minutes (4.0 * 60.0)
+    private static let MaxTeamVideoLength: TimeInterval = 960 // 16 minutes (16.0 * 60.0)
+
+    private var selfUserHasTeam: Bool {
+        return self.selfUser.hasTeam
+    }
+
+    public var maxUploadFileSize: UInt64 {
+        return UInt64.uploadFileSizeLimit(hasTeam: selfUserHasTeam)
+    }
+
+    public var maxAudioLength: TimeInterval {
+        return selfUserHasTeam ? ZMUserSession.MaxTeamAudioLength : ZMUserSession.MaxAudioLength
+    }
+
+    public var maxVideoLength: TimeInterval {
+        return selfUserHasTeam ? ZMUserSession.MaxTeamVideoLength : ZMUserSession.MaxVideoLength
+    }
+}
+
+extension UInt64 {
+    private static let MaxFileSize: UInt64 = 26214400 // 25 megabytes (25 * 1024 * 1024)
+    private static let MaxTeamFileSize: UInt64 = 104857600 // 100 megabytes (100 * 1024 * 1024)
+
+    public static func uploadFileSizeLimit(hasTeam: Bool) -> UInt64 {
+        return hasTeam ? MaxTeamFileSize : MaxFileSize
+    }
```

Comment:
This was copied over from the ui project, was it also deleted there?

Reply:
Just did!